viewer/packages/camera-manager/src/ComboControls.ts

```diff
@@ -202,6 +202,13 @@ export class ComboControls extends EventDispatcher {
    * Sets the enabled state of these controls.
    */
   set enabled(enabled: boolean) {
+    if (enabled === true && this._enabled === false) {
+      this.addEventListeners();
+    }
+    if (enabled === false && this._enabled === true) {
+      this.removeEventListeners();
+    }
```

Comment:
Btw, from a readers perspective this looks absolutly crazy. If it is enabled and also not enabled do something, and if it isn't enabled but also enabled do something else?

```suggestion
    if (enabled && !this._enabled) {
      this.addEventListeners();
    }
    if (!enabled && this._enabled) {
      this.removeEventListeners();
    }
```

Reply:
We can rename function argument to something like `newEnabledState` or `newState`.